checks.d/etcd.py

```diff
@@ -68,6 +68,17 @@ def check(self, instance):
         # Load values from the instance config
         url = instance['url']
         instance_tags = instance.get('tags', [])
+
+        ssl_params = {
+            'ssl': instance.get('ssl', False),
```

Comment:
Also should SSL be enabled by default when using a `https` scheme? Or can we maybe get rid of this boolean and just check for that.